schema/TRIBAL_NATIONAL_PARTY_CONVENTION_ACCOUNT.json

```diff
@@ -0,0 +1,413 @@
+                "TYPE": "NUM-8",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": 20120615,
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": null,
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "contribution_amount": {
+            "title": "CONTRIBUTION AMOUNT",
+            "description": "",
+            "type": "number",
+            "minimum": -99999999.99,
+            "maximum": 999999999.99,
+            "examples": [
+                250
+            ],
+            "fec_spec": {
+                "FIELD_DESCRIPTION": "CONTRIBUTION AMOUNT",
+                "TYPE": "AMT-12",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": 250,
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": null,
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "contribution_aggregate": {
+            "title": "CONTRIBUTION AGGREGATE",
+            "description": "",
+            "type": "number",
+            "minimum": -99999999.99,
+            "maximum": 999999999.99,
+            "examples": [
+                1000
+            ],
+            "fec_spec": {
+                "FIELD_DESCRIPTION": "CONTRIBUTION AGGREGATE",
+                "TYPE": "AMT-12",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": 1000,
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": "YTD",
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "aggregation_group": {
+            "title": "AGGREGATION GROUP",
+            "description": "",
+            "const": "NATIONAL_PARTY_CONVENTION_ACCOUNT",
+            "examples": [
+                "NATIONAL_PARTY_CONVENTION_ACCOUNT"
+            ],
+            "fec_spec": {
+                "FIELD_DESCRIPTION": "AGGREGATION GROUP",
+                "TYPE": "A/N-100",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": "NATIONAL_PARTY_CONVENTION_ACCOUNT",
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": null,
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "contribution_purpose_descrip": {
+            "title": "CONTRIBUTION PURPOSE DESCRIPTION",
+            "description": "",
+            "const": "Pres. Nominating Convention Account",
+            "examples": [
+                "Pres. Nominating Convention Account"
+            ],
+            "fec_spec": {
+                "FIELD_DESCRIPTION": "CONTRIBUTION PURPOSE DESCRIPTION",
+                "TYPE": "A/N-100",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": "Pres. Nominating Convention Account",
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": "Pres. Nominating Convention Account",
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
```

Comment:
Missing contributor_employer and contributor_occupation properties.

Reply:
Oops, good catch.  I've updated this to include the missing field properties